pkg/cmd/pipelinerun/describe.go

```diff
@@ -97,6 +97,11 @@ func printPipelineRunStatus(w *tabwriter.Writer, pr *v1alpha1.PipelineRun, p cli
 	fmt.Fprintf(w, statusBody, formatted.Age(*pr.Status.StartTime, p.Time()),
 		formatted.Duration(pr.Status.StartTime, pr.Status.CompletionTime),
 		formatted.Condition(pr.Status.Conditions[0]))
+
+	if (formatted.IsFailed(pr.Status.Conditions[0])) {
+		fmt.Fprintln(w, "\nMessage")
+		fmt.Fprintln(w, pr.Status.Conditions[0].Message)
```

Comment:
what if message field is empty?
Is there any probability? :thinking:

Reply:
I feel there is :sweat_smile: